### PR TITLE
Import standing data

### PR DESCRIPTION
**Pull request by monotypical**
These files have been copied from the main Bichard7 repo
See https://github.com/ministryofjustice/bichard7-next/tree/1d2f36cd179e8ec56e04ce7f74e4513611c2842d/bichard-backend/src/main/resources/config-data

### CODE REVIEW

File: output-data/alcohol-level-method.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "description": "Benzoylecgonine",
+    "spiCode": "D"
+  },
+  {
+    "cjsCode": "E",
+    "description": "Clonazepam",
+    "spiCode": "E"
+  },
+  {
+    "cjsCode": "F",
```
Comment (tomvaughan77) [inline output-data/alcohol-level-method.json:28]: Are we sure these are all `alcohol-level-method`s?

File: output-data/amount.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+[
```
Comment (tomvaughan77) [inline output-data/amount.json:1]: `amount` of what? Name of this file is a bit odd given the contents

**Review by daviesjamie — APPROVED** [review]

**Review by tomvaughan77 — APPROVED** [review]

File: output-data/amount.json (hunk 2 above)
Comment (tomvaughan77) [reply output-data/amount.json:1]: Naming came directly from `Bichard`- ignoring

File: output-data/alcohol-level-method.json (hunk 1 above)
Comment (tomvaughan77) [reply output-data/alcohol-level-method.json:28]: Naming came directly from `Bichard`- ignoring

Comment (monotypical) [reply output-data/alcohol-level-method.json:28]: we're just copying the existing data structure for now

**Pull request merged**